### PR TITLE
Reorientation settings

### PR DESCRIPTION
**Pull request by MiszczuZPolski**
**When merged this pull request will add:**
- option for enabling/disabling slap sound,
- option for enabling/disabling reorientation,
- option for medical level required for reorientation.

### CODE REVIEW

**Comment by YetheSamartaka** [comment]
LGTM.

**Comment by MiszczuZPolski** [comment]
Added "Unconscious SpO2 value" to settings

File: addons/pharma/stringtable.xml [hunk 1]
```diff
@@ -911,5 +911,83 @@
...
+            <Korean>방향 전환이 활성화되었습니까?</Korean>
+            <French>Stimulation activé?</French>
+            <Turkish>Yeniden yönlendirme etkin mi?</Turkish>
+            <Italian>Stimolazione abilitata?</Italian>
+            <Japanese>患者に刺激を与える 機能を有効化しますか?</Japanese>
+        </Key>
+        <Key ID="STR_KAT_circulation_SETTING_Reorientation_Enable_DESC">
+            <English>Enables reorientation</English>
+            <Polish>Pozwala na cucenie</Polish>
+            <German>Aktiviert die Umorientieren</German>
```
Comment (Tomcat-SG) [inline addons/pharma/stringtable.xml]:
```suggestion
            <German>Aktiviert das Umorientieren</German>
```

File: addons/pharma/stringtable.xml [hunk 2]
```diff
@@ -911,5 +911,83 @@
...
+            <Korean>방향 변경 허용</Korean>
+            <French>Autoriser stimulation</French>
+            <Turkish>Yeniden yönlendirmeye izin ver</Turkish>
+            <Italian>Autorizza stimolazione</Italian>
+            <Japanese>患者に刺激を与えることが出来る人</Japanese>
+        </Key>
+        <Key ID="STR_KAT_circulation_SETTING_Allow_Reorientation_DESC">
+            <English>Medical level required for reorientation</English>
+            <Polish>Poziom wyszkolenia wymagany do cucenia</Polish>
+            <German>Medizinisches Niveau für Umorientieren erforderlich</German>
```
Comment (Tomcat-SG) [inline addons/pharma/stringtable.xml]:
```suggestion
            <German>Erforderliches Medic Level zum Umorientieren</German>
```

**Review by Tomcat-SG — CHANGES_REQUESTED** [review]

**Review by Tomcat-SG — APPROVED** [review]
Now LGTM

**Pull request merged**